docs/deployment/hosting-repositories.rst

```diff
@@ -68,6 +68,15 @@ can be filled to the specified quota.
 If storage quotas are used, ensure that all deployed Borg releases
 support storage quotas.
 
+**Append-only repositorues**
```

Comment:
typo

Reply:
also, compared to the paragraph above, which has some introduction into usage, this one is only pointing out the special stuff.

Reply:
the 2nd item is still todo.

Reply:
What do you mean exactly ?
The former paragraph is about storage quotas, which seems also a special stuff, no?

Reply:
the paragraph about quotas first tells general stuff, like how to use them.

this paragraph does not, it just says what does not work like some people could maybe expect.

Reply:
i'll change the remaining stuff later.